quicly-probes.d

```diff
@@ -20,6 +20,24 @@
  * IN THE SOFTWARE.
  */
 
+/* @appdata
+{
+    "receive": ["bytes"],
```

Comment:
I do not think we need this. `bytes` of the receive probe carry bytes transmitted on the wire. Logging them is no more harmful than doing a packet capture.